geometry_msgs/Point center
uint32 width
uint32 height
